### ISSUE TITLE
Introductory Data Transformation with dplyr (current Ch. 2) resources

### ISSUE THREAD

**Issue by apreshill** [dspatterns/dspatterns#7]
Chapter: https://edr-book.netlify.app/intro-transformation-dplyr.html

Challenge to organize (see: https://moderndive.com/3-wrangling.html)

May want too loop in tables earlier here to give a big "payoff"

Possible project:
https://apreshill.github.io/data-vis-labs-2018/02-moma.html

Slides:
+ https://apreshill.github.io/data-vis-labs-2018/slides/02-slides.html#1
+ https://apreshill.github.io/data-vis-labs-2018/slides/03-slides.html#1

**Comment by rich-iannone** [comment]
I agree that tables early on would provide a better payoff than the console output. Combined with R Markdown, this could show how create a nice in-lesson deliverable and a way to teach some more R Markdown basics.

Thanks for getting a project idea in here. Seems like the first two chapters should be worked on first. They could set us up with a nice format for getting the rest of them reworked.

**Comment by rich-iannone** [comment]
These resources will be used in Chapter 4 (for the project).
